x-pack/plugins/ml/public/application/components/field_stats_flyout/field_stats_flyout_provider.tsx

```diff
@@ -27,6 +47,68 @@ export const FieldStatsFlyoutProvider: FC<{
     () => setFieldStatsIsFlyoutVisible(!isFieldStatsFlyoutVisible),
     [isFieldStatsFlyoutVisible]
   );
+  const [manager] = useState(new PopulatedFieldsCacheManager());
+  const [populatedFields$] = useState(new BehaviorSubject<Set<string>>(new Set()));
+
+  useEffect(() => {
+    if (disablePopulatedFields) return;
+    const abortController = new AbortController();
```

Comment:
If you invoke this effect consecutively, while previous promise hasn't been resolved yet, you need to abort it. 
I suggest keeping a reference of the AbortController instance (with `useRef`), and abort it at the beginning of this callback.

Reply:
Updated here [`2dcb364` (#163371)](https://github.com/elastic/kibana/pull/163371/commits/2dcb3644dc0702612c4d46b5e44523f04a1c865b)